debian/patches/disable-updates.patch

```diff
@@ -0,0 +1,11 @@
+--- etcher-1.0.0~beta.15.orig/lib/gui/app.js
++++ etcher-1.0.0~beta.15/lib/gui/app.js
+@@ -65,7 +65,7 @@ const app = angular.module('Etcher', [
+ app.run((AnalyticsService, UpdateNotifierService, SelectionStateModel) => {
+   AnalyticsService.logEvent('Application start');
+ 
+-  if (UpdateNotifierService.shouldCheckForUpdates() && !process.env.ETCHER_DISABLE_UPDATES) {
++  if (false && UpdateNotifierService.shouldCheckForUpdates() && !process.env.ETCHER_DISABLE_UPDATES) {
```

Comment:
We allow disabling update notifications using the `ETCHER_DISABLE_UPDATES` environment variable (see https://github.com/resin-io/etcher/blob/master/docs/USER-DOCUMENTATION.md#disable-update-notifications). Is there a way we can re-use that instead of patching directly? I'm thinking that you could provide a script that calls the Etcher executable setting that env var first. Happy to provide such option in another form that helps you.